### PR TITLE
Update restart commands for 3.10

### PR DESCRIPTION
**Pull request by mburke5678**
https://trello.com/c/u05F8OC3/18-new-command-to-restart-services

### CODE REVIEW

**Comment by mburke5678** [comment]
@sdodson 

Couple of questions, if I may:
1. I saw different restart commands for etcd. Do these commands to different things?
 `systemctl restart etcd.service` and `systemctl restart etcd`

2. Currently, there are different restart command for Origin. In 3.10+ will there be separate commands for Enterprise and Origin?

3. I see a few instances of separate restart commands for "a single master cluster installation"
For example: https://docs.openshift.com/container-platform/3.9/day_two_guide/host_level_tasks.html#removing-an-etcd-host 
Does multiple vs single master make a difference with the new commands?

4. What about the `restart atomic-openshift-node` command? Is that changing?

**Comment by sdodson** [comment]
> I saw different restart commands for etcd. Do these commands to different things?
> systemctl restart etcd.service and systemctl restart etcd

These two are effectively the same, etcd.service is the more proper name for the systemd unit, but both will work.



> Currently, there are different restart command for Origin. In 3.10+ will there be separate commands for Enterprise and Origin?

No, `master-restart` is uniform across origin and ocp.

> I see a few instances of separate restart commands for "a single master cluster installation"
> For example: https://docs.openshift.com/container-platform/3.9/day_two_guide/host_level_tasks.html#removing-an-etcd-host
> Does multiple vs single master make a difference with the new commands?

I'd get rid of the references to single master, those went away after upgrading to 3.7.

> What about the restart atomic-openshift-node command? Is that changing?

No, this remains the same and it's different between origin and ocp as it was before `systemctl restart origin-node` vs `systemctl restart atomic-openshift-node`

File: admin_guide/disabling_features.adoc [hunk 1]
```diff
@@ -77,17 +77,9 @@ kubernetesMasterConfig:
 
 . Restart the {product-title} master service to apply the changes.
 +
-ifdef::openshift-enterprise[]
-----
-# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
-endif::[]
-ifdef::openshift-origin[]
+# systemctl master-restart api master-restart controllers
```
Comment (vikram-redhat) [inline admin_guide/disabling_features.adoc]: @mburke5678 - I am guessing that this is correct? That it is the same command now for both origin and enterprise and that is why the ifdefs are removed?

File: admin_guide/encrypting_data.adoc [hunk 2]
```diff
@@ -276,33 +276,19 @@ on all servers.
...
 +
-ifdef::openshift-origin[]
-----
-# systemctl restart origin-master-api origin-master-controllers
-----
-endif::[]
-ifdef::openshift-enterprise[]
 ----
-# systemctl restart atomic-openshift-master-api
+# systemctl master-restart api
```
Comment (vikram-redhat) [inline admin_guide/encrypting_data.adoc]: @mburke5678 Again want to make sure that both origin and enterprise now share the command?

File: admin_guide/disabling_features.adoc (hunk 1 above)
Comment (mburke5678) [reply admin_guide/disabling_features.adoc]: @vikram-redhat Yes. For controller (master), API, and etcd, the restart commands are the same for enterprise and origin. But the node restart commands are still different for each.
See [Scott's comment above](https://github.com/openshift/openshift-docs/pull/10293#issuecomment-399206510).

**Comment by mburke5678** [comment]
@sdodson How does this look for QE review?

**Comment by mburke5678** [comment]
@ganhuang PTAL. 
This PR addresses only the restart commands in https://trello.com/c/u05F8OC3/18-new-command-to-restart-services.
The logging commands are covered in a separate PR that is not ready for review at the time I posted this comment. 
https://github.com/openshift/openshift-docs/pull/10313

File: admin_guide/backup_restore.adoc [hunk 3]
```diff
@@ -387,7 +387,7 @@ and displays {product-title}'s configuration:
 +
 ----
 # systemctl daemon-reload
-# systemctl restart etcd
+# systemctl master-restart etcd
```
Comment (geliu2016) [inline admin_guide/backup_restore.adoc]: systemctl is redundant, the command line running result is: 
# systemctl master-restart etcd
Unknown operation 'master-restart'.
so master-restart can't work with systemctl, the correct command line is:  master-restart etcd, similar for 'api' and 'controller' in other place in this doc.

**Comment by mburke5678** [comment]
@geliu2016 I saw an email with a comment from you that I do not see in the PR. I made the changes suggested in that email:
In admin_guide/backup_restore.adoc:

> @@ -387,7 +387,7 @@ and displays {product-title}'s configuration:
 +
-# systemctl daemon-reload
-# systemctl restart etcd
+# systemctl master-restart etcd

systemctl is redundant, the command line running result is:
systemctl master-restart etcd

Unknown operation 'master-restart'.
so master-restart can't work with systemctl, the correct command line is: master-restart etcd, similar for 'api' and 'controller' in other place in this doc.

**Comment by mburke5678** [comment]
@geliu2016 PTAL ^^

**Comment by mburke5678** [comment]
Docs PR set to Accepted.  https://trello.com/c/nzYUisjF/902-new-command-to-restart-services#comment-5b3b1aa346b9d18951eed460
@openshift/team-documentation PTAL

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: admin_guide/out_of_resource_handling.adoc [hunk 4]
```diff
@@ -43,7 +43,7 @@ An _eviction policy_ allows a node to fail one or more pods when the node is run
 Failing a pod allows the node to reclaim needed resources.
 
 An eviciton policy is a combination of an xref:out-of-resource-eviction-signals[eviction trigger signal] with a specific xref:out-of-resource-eviction-thresholds[eviction threshold value], that is set
-in the xref:out-of-resource-eviction-policy[node configuration file] or through the xref:out-of-resource-eviction-thresholds[command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. See the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
+in the node configuration file or through the xref:out-of-resource-eviction-thresholds[command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. See the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
```
Comment (kalexand-rh) [inline admin_guide/out_of_resource_handling.adoc]: s/sections below/following sections
s/information the/information about the

**Review by kalexand-rh — COMMENTED** [review]
Is this change correct in dedicated? Some of these commands are filtering up to that collection. The if statement changes look good. Aside from a nitpick and the question about dedicated, this LGTM.

File: admin_guide/out_of_resource_handling.adoc (hunk 4 above)
Comment (mburke5678) [reply admin_guide/out_of_resource_handling.adoc]: @kalexand-rh Which change are you asking about, related to Dedicated. The paragraph above or the PR in general? I did a search on `systemctl restart` in Dedicated (via the browser) and got no results. I can verify that all of the topics in this PR are not in Dedicated if you think there is a concern.

Comment (kalexand-rh) [reply admin_guide/out_of_resource_handling.adoc]: In general. For example, the command in admin_guide/managing_projects.adoc is visible in the dedicated version of this built file, and I'm not sure if it should be.

Comment (mburke5678) [reply admin_guide/out_of_resource_handling.adoc]: @kalexand-rh 
It appears that, of the files in the PR, only the Admin Guide -> Managing Projects and Setting Quotas topics are in Dedicated. Both appear to be properly ifdef'd to keep the restart commands out. Where are you seeing the commands in Managing Projects?

Comment (mburke5678) [reply admin_guide/out_of_resource_handling.adoc]: @kalexand-rh I moved the endif to the end of the file. Page appears correct in Asciibinder preview

**Pull request merged**